React/Views/RCTView.h

```diff
@@ -48,7 +48,7 @@ typedef void (^RCTViewEventHandler)(RCTView *view);
 
 /**
  * Hide subviews if they are outside the view bounds.
- * This is an optimisation used predominantly with RKScrollViews
+ * This is an optimisation used predominantly with RCTScrollViews
```

Comment:
nit: I know it was pluralized in the original, but this would read better as `RCTScrollView`